server/src/main/resources/reference.conf

```diff
@@ -1,23 +1,19 @@
 {
-   logger {
-      # This format includes timestamp, level, thread (fiberId), message, and cause
-      format = "%label{timestamp}{%fixed{32}{%timestamp}} %label{level}{%level} %label{thread}{%fiberId} %label{class}{%name:%line} %label{message}{%message} %label{cause}{%cause}"
-      # log filter
-      filter {
-         # rootLevel sets the minimum level of log messages that will be displayed
-         rootLevel = INFO
```

Comment:
is this not used at all? I made an assumption that it's loaded thanks to this:

`private val configProvider: ConfigProvider = TypesafeConfigProvider.fromResourcePath()`

Reply:
I have replaced this logging config section by logback.xml as I have noticed the mappings were not working so we could configure only log level of our own classes. Now there is SLF4s used with logback logging backend. We can talk more about it on a call if you want.